### PR TITLE
add basic support files for rtl87x2g series soc

### PR DESCRIPTION
**Pull request by ZhiyuanTang17**
In this PR, I add code for supporting the realtek rtl87x2g series soc. 
With this PR, you can build and run a hello_world app with rtl8762gn_evb.
Hi @RSGraceGuan @YuzhuoLiuRTK , Please help to review this PR.

### CODE REVIEW

**Review by YuzhuoLiuRTK — APPROVED** [review]

File: include/zephyr/dt-bindings/clock/rtl87x2g-clocks.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#define APBPeriph_KEYSCAN_CLOCK            24U
+#define APBPeriph_24BADC_CLOCK             25U
+#define APBPeriph_ADC_CLOCK                26U
+#define APBPeriph_CAN_CLOCK                27U
+#define APBPeriph_IR_CLOCK                 28U
+#define APBPeriph_ISO7816_CLOCK            29U
+#define APBPeriph_GPIOB_CLOCK              30U
+#define APBPeriph_GPIOA_CLOCK              31U
+#define APBPeriph_DISP_CLOCK               32U
+#define APBPeriph_RTZIP_CLOCK              33U
```
Comment (YuzhuoLiuRTK) [inline include/zephyr/dt-bindings/clock/rtl87x2g-clocks.h]: rename to APBPeriph_IMDC_CLOCK

Comment (ZhiyuanTang17) [reply include/zephyr/dt-bindings/clock/rtl87x2g-clocks.h]: I have pushed a commit to solve this problem.

**Review by YuzhuoLiuRTK — APPROVED** [review]

**Comment by ZhiyuanTang17** [comment]
Hi @RSGraceGuan ， I have updated a commit to rename RTK to REALTEK. Please help to review, thank you.

**Comment by ZhiyuanTang17** [comment]
> Hi @RSGraceGuan ， I have updated a commit to rename RTK to REALTEK. Please help to review, thank you.

This PR will be closed. Because @YuzhuoLiuRTK  has a new PR which will contains the commits in this PR.

**Pull request closed without merging**